### ISSUE TITLE
Add option to select photos from Google Photos and other sources

### ISSUE THREAD

**Issue by SreenuAndroid** [esafirm/android-image-picker#120]
Currently, the library supports only local photos in Explorer. This makes the library inefficient and incomplete for bigger projects. Please add option to select photos from other sources like Google Photos and other remote options like Drive, Dropbox and so on.

**Comment by esafirm** [comment]
You're right, but AFAIK we can't show / query the images from Google photos or other remote sources. Any idea about the UI and UX?

**Comment by india2sarthak** [comment]
Hey @esafirm, I think we can have an "other" option the folder activity. Which on clicking would open the documents picker of the system, which would show the photos present in Google Photos, Drive, Dropbox etc which users could select from. I've seen Instagram and Twitter implement a similar architecture to achieve this. Let me know your thoughts.

**Comment by india2sarthak** [comment]
Okay, so I created a PR (https://github.com/esafirm/android-image-picker/pull/400) with the same capability. @esafirm Please do take a look at it!
